### DISCUSSION TITLE
ReadMe: Tile list entry options

### DISCUSSION THREAD

**Discussion by Masteroshi430** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#148] [Q&A] [answered]
Sorry but it isn't clear in the ReadMe :
I'm trying to configure Tile list entry options so I assumed I should enter: 

```
tile_list_entry:
  label: Mop left
  entity: input_number.mop_left
```
but it doesn't work.
please add an example in the ReadMe 😊

EDIT: it's apparently tiles: but I have that error : (intermediate value)(intermediate value)(intermediate value).flatMap is not a function

**Comment by PiotrMachowski** [comment, marked as answer]
@Masteroshi430 what is your config? it should look like this:

```yaml
...
tiles:
  - label: Mop left
    entity: input_number.mop_left
```

**Reply by Masteroshi430** [reply]
ah yes it was missing the minus sign! 
thank you!

**Comment by Flavio5** [comment]
Does is possibile to add a new tile keeping the tiles already autogenerated?

**Reply by PiotrMachowski** [reply]
@Flavio5 yes, this feature has been added in v2.0.2: [`appand_tiles: true`](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#preset-options)

**Reply by Photelegy** [reply]
How would the code for the autogenerated one look?
With the Icons, Text, ...
So I can edit it.
My idea is to only show "Status", "Battery" and "Fan speed" to save space.

![image](https://user-images.githubusercontent.com/61714583/172160524-07da9e71-72a7-4742-b461-2d4daf026aad.png)

**Reply by PiotrMachowski** [reply]
@Photelegy https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/321

**Reply by Photelegy** [reply]
Perfect!
Thank you!
